### DISCUSSION TITLE
Updates from two locations

### DISCUSSION THREAD

**Discussion by mattmazer0397** [WhitewaterFoundry/Pengwin#680] [Q&A]
I just had a quick question. I typically use "pengwin-setup update" to check for and download updates. However, I also occasionally get updates through the Windows Store. Why is this?

**Comment by crramirez** [comment]
Hello @mattmazer0397,

When you use pengwin-setup update, you are updating your installed distribution. It is the live update.

When you get updates through Windows Store, it is updating the base image. In case that you delete your distro and start from scratch, you will start with the latest update. This avoids you to make a huge update after you install the distro.

Tell me if it is clear.

I will move this issue to discussions because it makes more sense there.

Regards,
Carlos
